config/initializers/salesforce_services.rb

```diff
@@ -0,0 +1,6 @@
+### shortcut to alias SalesforceServices
+# NOTE: if you update any of these services (new methods, etc),
+#       you will need to restart spring in order for the alias to "refresh"
+AccountService = SalesforceService::AccountService
```

Comment:
cool i actually might need to update my pr since i declare this in the devise overrides registrations ctrl